### PR TITLE
Guide and minor main page changes for 1.1.0

### PR DESCRIPTION
**Pull request by themantissa**
## Description

1.1.0 Info and upgrade guide updates before release and a few minor index page changes based on recent user feedback.

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ X] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 1]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+* Projects created with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)` can now have a default owner set with `project_owner_id`
+* [`mongodbatlas_privatelink_endpoint`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint) and [`mongodbatlas_privatelink_endpoint_service`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service) now support Google Cloud'd Private Service Connect
+* Projects created with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)` can now have default alert creation disabled by setting `with_default_alerts_settings` to false.
+* [`mongodbatlas_search_index`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_index) now supports synonyms.
+* Atlas Serverless deployments, now in preview, can now be created with [`mongodbatlas_serverless_instance`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_instance)
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]: passive -> active voice, periods at the end of list items that are complete sentences per [style guide](https://docs.mongodb.com/meta/style-guide/style/lists/#list-items), and minor updates for clarity
```suggestion
* You can now manage multi-cloud deployments with [`mongodbatlas_advanced_clusters`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_clusters).
* You can now manage invitations with
[`mongodbatlas_org_invitation`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/org_invitation) and [`mongodbatlas_project_invitation`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project_invitation).
* You can now use `project_owner_id` to set a default owner of projects that you create with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project).
* [`mongodbatlas_privatelink_endpoint`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint) and [`mongodbatlas_privatelink_endpoint_service`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service) now support private endpoints on Google Cloud with Private Service Connect.
* You can now set `with_default_alerts_settings` to `false` to disable the default alerts when you create projects with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project).
* [`mongodbatlas_search_index`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_index) now supports synonyms.
* You can now create Atlas Serverless deployments (available as a preview feature) with [`mongodbatlas_serverless_instance`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_instance).
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 2]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
 
-1. Remove your existing `mongodbatlas_cloud_provider_snapshot` resources from terraform state
-   **NOTE** before doing any terraform state change please do a backup
+## Migration from Cloud Provider Snapshot to Cloud Backup Snapshot
 
-```bash
-$ terraform state rm mongodbatlas_cloud_provider_snapshot.test
-```
+The term Cloud Provider has been deprecated in favor of the term Cloud Backup.  In order to support this change within the Terraform provider, related resources and data sources with the older terminology have been deprecated and new resources and data sources added to support the new terminology.  In order to upgrade from the previous resources to the new you'll need to select from one of the following two options:
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]: bold to emphasize terms
```suggestion
The term **Cloud Provider** has been deprecated in favor of the term **Cloud Backup**.  In order to support this change within the Terraform provider, related resources and data sources with the older terminology have been deprecated and new resources and data sources have been added to support the new terminology.  In order to upgrade from the previous resources to the new you'll need to select from one of the following two options:
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 3]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
 
-2. Change the version of provider to the v1.1.0
+### Option 1: Replace the name of resource in terraform state
 
-3. In order to import the new resource, you must first change your Terraform configuration file, replacing `mongodbatlas_cloud_provider_snapshot_backup_policy` with `mongodbatlas_cloud_backup_schedule`. See the example shared above or find more information in the [mongodbatlas_cloud_backup_schedule](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule)
+1. Replace the name of the deprecated provider `mongodbatlas_cloud_provider_snapshot` with the new name `mongodbatlas_cloud_backup_snapshot` in the Terraform state file (`terraform.tfstate`).  Do the same for `mongodbatlas_cloud_provider_snapshot_restore_job` and `mongodbatlas_cloud_backup_snapshot_restore_job`
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
1. Replace the name of the deprecated resource `mongodbatlas_cloud_provider_snapshot` with the new name `mongodbatlas_cloud_backup_snapshot` in the Terraform state file (`terraform.tfstate`).  Do the same for `mongodbatlas_cloud_provider_snapshot_restore_job` and `mongodbatlas_cloud_backup_snapshot_restore_job`.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 4]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+   **NOTE** Before performing any Terraform state change please backup your state.
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
   **NOTE** Before performing any Terraform state change, please backup your state.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown (hunk 2 above)
Comment (jwilliams-mongo) [reply website/docs/guides/1.1.0-upgrade-guide.html.markdown]: have these been deprecated or removed? Based on the steps below, looks like the old providers have been removed.

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 5]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
 
-
 No changes. Your infrastructure matches the configuration.
 
 Terraform has compared your real infrastructure against your configuration and found no differences,
 so no changes are needed.
 ```
 
-### Option 2: Replace the name of resource in terraform state
+### Option 1:  Remove the existing and import the new resource
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]: choose deprecated or removed based on if the old resources are removed
```suggestion
### Option 2:  Remove the deprecated|removed and import the new resources
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 6]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+* Projects created with [`mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)` can now have default alert creation disabled by setting `with_default_alerts_settings` to false.
+* [`mongodbatlas_search_index`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/search_index) now supports synonyms.
+* Atlas Serverless deployments, now in preview, can now be created with [`mongodbatlas_serverless_instance`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_instance)
 
-There are two possible options to upgrade from v1.0.2 to v1.1.0 from the existing resources `mongodbatlas_cloud_provider_snapshot_backup_policy` to the new ones
+Changes:
+In addition, there are changes that will require modifications to your Terraform configuration between earlier versions of the provider and this one.
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]: The bulk of these changes are covered in the next section. I think it's appropriate to link there.

consider something like:

Version 1.10 removes|deprecates the following resources:

- [`mongodbatlas_cloud_provider_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot)
- [`mongodbatlas_cloud_provider_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job)

If your Terraform configuration uses these resources, see [Migration from Cloud Provider Snapshot to Cloud Backup Snapshot](#migration-from-cloud-provider-snapshot-to-cloud-backup-snapshot) to migrate to the new resources **before you upgrade**.

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 7]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
 
-3. In order to import the new resource, you must first change your Terraform configuration file, replacing `mongodbatlas_cloud_provider_snapshot_backup_policy` with `mongodbatlas_cloud_backup_schedule`. See the example shared above or find more information in the [mongodbatlas_cloud_backup_schedule](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule)
+1. Replace the name of the deprecated provider `mongodbatlas_cloud_provider_snapshot` with the new name `mongodbatlas_cloud_backup_snapshot` in the Terraform state file (`terraform.tfstate`).  Do the same for `mongodbatlas_cloud_provider_snapshot_restore_job` and `mongodbatlas_cloud_backup_snapshot_restore_job`
+   **NOTE** Before performing any Terraform state change please backup your state.
 
-4. Perform an import of your existing `mongodbatlas_cloud_provider_snapshot` to `mongodbatlas_cloud_backup_snapshot`
-```bash
-    terraform import mongodbatlas_cloud_backup_snapshot.test <project_id>-<cluster_name>-<snapshot_id>
-```
+2. Upgrade the provider to v1.1.0
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
2. Upgrade the provider to v1.1.0.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 8]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
 
-5. Do a terraform plan (no changes detected)
+3. Run a `terraform plan` to ensure no changes are detected
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
3. Run a `terraform plan` to ensure no changes are detected.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 9]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+4. Import your existing `mongodbatlas_cloud_provider_snapshot` to `mongodbatlas_cloud_backup_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` to `mongodbatlas_cloud_backup_snapshot_restore_job`
+```bash
+    terraform import mongodbatlas_cloud_backup_snapshot.test <project_id>-<cluster_name>-<snapshot_id>
+```
+```bash
+    terraform import mongodbatlas_cloud_backup_snapshot_restore_job.test <project_id>-<cluster_name>-<job_id>
+```
 
-3. Do a terraform plan (no changes detected)
+5. Run a `terraform plan` to ensure no changes are detected
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
5. Run a `terraform plan` to ensure no changes are detected.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 10]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+```
+
+2. Upgrade the provider to v1.1.0
 
-1. Replace then name of the state in your existing `mongodbatlas_cloud_provider_snapshot` with `mongodbatlas_cloud_provider_snapshot` in terraform state file (`terraform.tfstate`).
-   **NOTE** before doing any terraform state change please do a backup
+3. Change your Terraform configuration file, replacing `mongodbatlas_cloud_provider_snapshot` with ``mongodbatlas_cloud_backup_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` with `mongodbatlas_cloud_backup_snapshot_restore_job`
 
-2. Change the version of provider to the v1.1.0
+4. Import your existing `mongodbatlas_cloud_provider_snapshot` to `mongodbatlas_cloud_backup_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` to `mongodbatlas_cloud_backup_snapshot_restore_job`
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
4. Import your existing `mongodbatlas_cloud_provider_snapshot` to `mongodbatlas_cloud_backup_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` to `mongodbatlas_cloud_backup_snapshot_restore_job`:
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 11]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
 
-1. Remove your existing `mongodbatlas_cloud_provider_snapshot` resources from terraform state
-   **NOTE** before doing any terraform state change please do a backup
+## Migration from Cloud Provider Snapshot to Cloud Backup Snapshot
 
-```bash
-$ terraform state rm mongodbatlas_cloud_provider_snapshot.test
-```
+The term Cloud Provider has been deprecated in favor of the term Cloud Backup.  In order to support this change within the Terraform provider, related resources and data sources with the older terminology have been deprecated and new resources and data sources added to support the new terminology.  In order to upgrade from the previous resources to the new you'll need to select from one of the following two options:
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown:38]: the difference between option 1 and option 2 is not clear to me. it looks like maybe if users have used the old providers, they must complete option 2. if they don't they can use option 1. is this accurate?

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 12]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
 
 Terraform has compared your real infrastructure against your configuration and found no differences,
 so no changes are needed.
 ```
 
-### Option 2: Replace the name of resource in terraform state
+### Option 1:  Remove the existing and import the new resource
+
+1. Remove your existing `mongodbatlas_cloud_provider_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` resources from the Terraform state file
+   **NOTE** Before performing any Terraform state change please backup your state.
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
1. Remove your existing `mongodbatlas_cloud_provider_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` resources from the Terraform state file.
   **NOTE** Before performing any Terraform state change, please backup your state.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 13]
```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+
+```bash
+$ terraform state rm mongodbatlas_cloud_provider_snapshot.test
+$ terraform state rm mongodbatlas_cloud_provider_snapshot_restore_job.test
+
+```
+
+2. Upgrade the provider to v1.1.0
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]:
```suggestion
2. Upgrade the provider to v1.1.0.
```

File: website/docs/index.html.markdown [hunk 14]
```diff
@@ -29,13 +29,16 @@ provider "mongodbatlas" {
 
 ## Configure Atlas Programmatic Access
 
-In order to set up authentication with the MongoDB Atlas provider a programmatic API key must be generated for MongoDB Atlas with the appropriate permissions and IP access list entries.
-The [MongoDB Atlas documentation](https://docs.atlas.mongodb.com/tutorial/manage-programmatic-access/index.html) contains the most up-to-date instructions for creating and managing your key(s) and IP access.
-Be aware, not all API resources require an IP access list by default, but one can set Atlas to require IP access entries for all API resources, see the [organization settings documentation](https://docs.atlas.mongodb.com/tutorial/manage-organization-settings/#require-ip-access-list-for-public-api) for more info.
+In order to set up authentication with the MongoDB Atlas provider a programmatic API key must be generated for MongoDB Atlas with the appropriate [role](https://docs.atlas.mongodb.com/reference/user-roles/) and IP access list entry.
```
Comment (jwilliams-mongo) [inline website/docs/index.html.markdown]: remove passive voice
```suggestion
In order to set up authentication with the MongoDB Atlas provider, you must generate a programmatic API key for MongoDB Atlas with the appropriate [role](https://docs.atlas.mongodb.com/reference/user-roles/) and IP access list entries.
```

File: website/docs/index.html.markdown [hunk 15]
```diff
@@ -29,13 +29,16 @@ provider "mongodbatlas" {
...
+The [MongoDB Atlas documentation](https://docs.atlas.mongodb.com/tutorial/manage-programmatic-access/index.html) contains the most up-to-date instructions for creating and managing your key(s), setting the appropriate role, and IP access.  
+
+Role: If unsure which role level to grant your key we suggest creating an Organization API Key with an Organization Owner role as that will ensure you have sufficient access for all actions.
```
Comment (jwilliams-mongo) [inline website/docs/index.html.markdown]: minor edits for clarity
```suggestion
**Role**: If unsure of which role level to grant your key, we suggest creating an organization API Key with an Organization Owner role. This ensures that you have sufficient access for all actions.
```

File: website/docs/index.html.markdown [hunk 16]
```diff
@@ -29,13 +29,16 @@ provider "mongodbatlas" {
...
+
+IP access list: Not all API resources require an IP access list by default but some do, such as backup, hence we highly suggest adding an IP access list as soon as possible.  See [Require IP Access List for Public API](https://docs.atlas.mongodb.com/tutorial/manage-organization-settings/#require-ip-access-list-for-public-api) for more info.
```
Comment (jwilliams-mongo) [inline website/docs/index.html.markdown]: minor edits for clarity
```suggestion
**IP access list**: Some API resources, such as backup resources, require an IP access list by default. We highly suggest that you add an IP access list as soon as possible.  See [Require IP Access List for Public API](https://docs.atlas.mongodb.com/tutorial/manage-organization-settings/#require-ip-access-list-for-public-api) for more info.
```

**Review by jwilliams-mongo — COMMENTED** [review]
@themantissa some suggestions for clarity and to remove passive voice. It's not clear to me if the old backup providers are being deprecated or removed in this release. It's also not clear when users should choose one or the other upgrade path. Users might benefit from some clarifications there.

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown (hunk 2 above)
Comment (themantissa) [reply website/docs/guides/1.1.0-upgrade-guide.html.markdown]: Just deprecated but obviously that's not clear.  Will adjust.

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown (hunk 5 above)
Comment (themantissa) [reply website/docs/guides/1.1.0-upgrade-guide.html.markdown]: Deprecated is accurate, so choosing that.

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown (hunk 6 above)
Comment (themantissa) [reply website/docs/guides/1.1.0-upgrade-guide.html.markdown]: Great suggestion!  I just added a clarification that they will be removed in a future major version.

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown (hunk 11 above)
Comment (themantissa) [reply website/docs/guides/1.1.0-upgrade-guide.html.markdown:38]: One requires just changing the names in the state file. Not a great practice in the Terraform world but works. The second is the more official way, but slightly harder.   It requires fully removing the old resources from state and then re-importing them.

**Comment by themantissa** [comment]
@jwilliams-mongo thank you so much.  Changes made. Sorry I forgot to batch the commits so kinda a mess!

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 17]
```diff
@@ -8,73 +8,90 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
+$ terraform state rm mongodbatlas_cloud_provider_snapshot.test
+$ terraform state rm mongodbatlas_cloud_provider_snapshot_restore_job.test
+
+```
+
+2. Upgrade the provider to v1.1.0.
 
-1. Replace then name of the state in your existing `mongodbatlas_cloud_provider_snapshot` with `mongodbatlas_cloud_provider_snapshot` in terraform state file (`terraform.tfstate`).
-   **NOTE** before doing any terraform state change please do a backup
+3. Change your Terraform configuration file, replacing `mongodbatlas_cloud_provider_snapshot` with ``mongodbatlas_cloud_backup_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` with `mongodbatlas_cloud_backup_snapshot_restore_job`
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown]: minor formatting and punctuation changes
```suggestion
3. Change your Terraform configuration file, replacing `mongodbatlas_cloud_provider_snapshot` with `mongodbatlas_cloud_backup_snapshot` and `mongodbatlas_cloud_provider_snapshot_restore_job` with `mongodbatlas_cloud_backup_snapshot_restore_job`.
```

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown [hunk 18]
```diff
@@ -8,73 +8,90 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
...
 
-The Terraform MongoDB Atlas Provider version 1.0.0 has a number of new and exciting features and changes.
+The Terraform MongoDB Atlas Provider version 1.1.0 has a number of new and exciting features and changes.
 
-* Cloud Backup Snapshot replaces the now deprecated Cloud Provider Snapshot.
-* Cloud Backup Snapshot Restore Job replaces the now deprecated Cloud Provider Snapshot Restore Job.
+New Features:
 
-1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
+* You can now manage multi-cloud deployments with [`mongodbatlas_advanced_clusters`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_clusters).
```
Comment (jwilliams-mongo) [inline website/docs/guides/1.1.0-upgrade-guide.html.markdown:15]: These links will work when this version is released, correct?

**Review by jwilliams-mongo — APPROVED** [review]
@themantissa LGTM % one small update. I checked the links -- they'll work when this version goes out, correct?

File: website/docs/guides/1.1.0-upgrade-guide.html.markdown (hunk 18 above)
Comment (themantissa) [reply website/docs/guides/1.1.0-upgrade-guide.html.markdown:15]: Correct, they won't work till the release goes out.

**Pull request merged**